audioread/rawread.py

```diff
@@ -130,7 +131,10 @@ def read_data(self, block_samples=1024):
             data = self._file.readframes(block_samples)
             if not data:
                 break
-
+            
+            remainder = len(data) % old_width 
+            if remainder != 0 :
+                data = data + PATCH_BYTE*(old_width-remainder)
```

Comment:
Here are some very low-level style (i.e., PEP8) comments:

- Please remove the whitespace on the blank line.
- Please remove the space between the 0 and the : in the `if` statement.
- Please *add* spaces around the binary operators `*` and `-`.